src/pages/Payment/components/PriceArea/PriceArea.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import AddPoints from '../AddPoints/AddPoints';
+import Button from '../../../../components/Button/Button';
+import './PriceArea.scss';
+import DetailsInfo from '../../../../components/DetailsInfo/DetailsInfo';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+const PriceArea = ({ points, userId, amount }) => {
+  const payAmount = Number(amount);
+  const userPoints = Number(points);
+  const navigate = useNavigate();
+  const [addPoints, setAddPoints] = useState('');
+  const [isOrder, setIsOrder] = useState(false);
+  const PAYMENTLABEL = [
+    { id: 1, type: '주문금액', content: payAmount },
+    { id: 2, type: '결제방법', content: '포인트사용' },
+    { id: 3, type: '보유포인트', content: userPoints },
+  ];
+
+  const handlePayment = () => {
+    axios
+      .post(
+        '백엔드결제api주소',
+        { amount: amount, points: points },
+        {
+          headers: {
+            // Authorization: `token`,
+            'Content-type': `application/json`,
+          },
+        },
+      )
+      .then(res => {
+        if (res.data.message === 'success') {
+          alert('결제 성공!');
+          navigate('/mypage');
+        }
+      });
+  };
+
+  useEffect(() => {
+    if (payAmount - userPoints > 0) {
+      setIsOrder(true);
+    }
+  }, [payAmount, userPoints]);
```

Comment:
- state를 만들 필요 없이, 조건 자체로 변수로 만들어 활용하시면 됩니다.
- 변수명도 명확하게 해주면 좋겠네요!
- falsy한 경우를 조건으로 하는 것보다는, truthy한 경우를 기반으로 사용하시는 게 덜 헷갈릴 것 같습니다.
```suggestion
  const isBuyable = userPoints >= payAmount;
```

Reply:
수정했습니다!